apps/artist/queries/overview.coffee

```diff
@@ -15,6 +15,14 @@ module.exports =
       artists (size: 15) @include(if: $artists){
         ... artistCell
       }
+      cta_artist: artists(size: 1) {
```

Comment:
Should rename this label to plural to reflect it containing an array (even if it only has 1 entry).